__init__.py

```diff
@@ -87,6 +67,7 @@ def __init__(self):
         self.flash_state = 0
         self.recurrence_dict = None
         self.sound_name = None
+        self.skill_control.category = 'system'
```

Comment:
I think using a "system" category for this skill could be confusing.  This gives us two different definitions of a "system" skill.  Could the category be something like "stateful" instead?  How do we see categories being used in skill control?

Reply:
What is a system skill? How are they currently delineated?

Categories are used to maintain backward compatibility with existing skills and to allow skills defined as category=system to have priority over non system skills during converse.